api/src/main/java/jakarta/servlet/jsp/jstl/tlv/package.html

```diff
@@ -42,7 +42,7 @@
   &lt;display-name&gt;permittedTaglibs&lt;/display-name&gt;
   &lt;tlib-version&gt;1.1&lt;/tlib-version&gt;
   &lt;short-name&gt;permittedTaglibs&lt;/short-name&gt;
-  &lt;uri&gt;http://java.sun.com/jstl/permittedTaglibs&lt;/uri&gt;
+  &lt;uri&gt;http://jakarta.apache.org/taglibs/standard/permittedTaglibs&lt;/uri&gt;
```

Comment:
Updated the example to use this URI instead.  Taken from the permittedTaglibs.tld : https://github.com/eclipse-ee4j/jstl-api/blob/master/impl/src/main/resources/META-INF/permittedTaglibs.tld#L30